src/DGtal/topology/Object.ih

```diff
@@ -891,6 +920,28 @@ DGtal::Object<TDigitalTopology, TDigitalSet>
   return geodesicN;
 }
 
+template <typename TDigitalTopology, typename TDigitalSet>
+inline
+bool
+DGtal::Object<TDigitalTopology, TDigitalSet>
+::isSimpleFromTable( const Point & center, const boost::dynamic_bitset<> & input_table, const std::unordered_map<Point,unsigned int> & pointMap ) const
+{
+  // Get Neighborhood Occupancy Configuration
+    using DomainConstIterator = typename Domain::ConstIterator;
+    Point p1 = Point::diagonal( -1 );
+    Point p2 = Point::diagonal(  1 );
+    Point c = Point::diagonal( 0 );
+    Domain ddomain( p1, p2 );
+    const auto & not_found( pointSet().end() );
+    unsigned int cfg{0};
```

Comment:
Perhaps you should define a type NeighborhoodConfiguration (in helpers/Neigh...) that is unsigned int and then use it where appropriate. I agree that it will be unsigned int32_t in all practical cases.